docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -297,6 +297,28 @@ variable to point to the installation of |hpx|.
    in your targets. The |hpx| include directories are available with the
    ``HPX_INCLUDE_DIRS`` CMake variable.
 
+.. _hpxcxx_documentation:
+
+Building |hpx| Components and Applications
+------------------------------------------
+The ``hpxcxx.in`` compiler wrapper in ``hpx/cmake/templates/`` directory helps to build a |hpx| component, an application or generate a ``<file-name>.o`` file, based on the arguments passed to it.
+
```

Comment:
@rstobaugh do we use the Oxford comma in our documentation? If yes, @MukundVaradarajan please change the sentence above accordingly.

Reply:
Or I can just use "or" everywhere and rephrase it as: "generate a ``<file-name>.o`` file or a hpx component or a hpx application". Would that be better?

Reply:
@hkaiser At least we did use the oxford comma in `docs/sphinx/manual/building_hpx.rst` 14 times with `, and `
@MukundVaradarajan I think we'd prefer the comma instead `component, an application, or generate a ...`

Reply:
@aurianer Got it. Will post the changes soon. Got caught up in some other work. Sorry, folks!

Reply:
@hkaiser @MukundVaradarajan There's no set rule for the Oxford comma that I'm aware of; as a whole, the manual tends not to use it. If you'd like to make a set rule either way, I can add that to our documentation style guide.

Reply:
@rstobaugh you are the English major ;-) Please make a decision and apply it.